doc/api/fs.md

```diff
@@ -151,6 +151,10 @@ Stop watching for changes on the given `fs.FSWatcher`.
 
 Emitted when the ReadStream's file is opened.
 
+### Event: 'close'
+
+Emitted when a ReadStream's file has ended or closed. Comes from fs.close() method. 
```

Comment:
Maybe something more specific like "Emitted when the ReadStream's underlying file descriptor has been closed" and similarly for `WriteStream`?